src/main/resources/application-integration-test.properties

```
@@ -18,3 +18,4 @@ com.ampnet.projectservice.file-storage.folder=test
 
```

Add grpc clinet wallet-service config:
```suggestion
grpc.client.wallet-service.address=static://wallet-service:8228
grpc.client.wallet-service.negotiationType=PLAINTEXT
```
See line 7.

---

Also add same config to application-dev.properties